src/main/java/com/uid2/admin/vertx/service/OperatorKeyService.java

```diff
@@ -53,16 +57,20 @@ public OperatorKeyService(JsonObject config,
                               RotatingOperatorKeyProvider operatorKeyProvider,
                               RotatingSiteStore siteProvider,
                               IKeyGenerator keyGenerator,
-                              KeyHasher keyHasher) {
+                              KeyHasher keyHasher,
+                              S3KeyManager s3KeyManager) {
         this.auth = auth;
         this.writeLock = writeLock;
         this.operatorKeyStoreWriter = operatorKeyStoreWriter;
         this.operatorKeyProvider = operatorKeyProvider;
         this.siteProvider = siteProvider;
         this.keyGenerator = keyGenerator;
         this.keyHasher = keyHasher;
+        this.s3KeyManager = s3KeyManager;
 
         this.operatorKeyPrefix = config.getString("operator_key_prefix");
+        this.s3KeyActivatesInSeconds = config.getLong("s3_key_activates_in_seconds");
```

Comment:
the getLong and getInteger have an overload that accepts a default value. It is probably worth using that here so that this does not break if the config hasn't been updated